guardian/pressreader
# Add CODE stage

## What does this change?

Adds config for a new CODE stage to our CDK and CI.

## How to test

Deploy and check whether a) the new resources are created, and b) the existing INFRA stage continues to work as it should.

## How can we measure success?

We are able to deploy changes to a test environment without risk to the production environment.

## Have we considered potential risks?

Adding this stage requires some changes to the existing INFRA stage config. Although I've tried to minimise the impact on INFRA, it's possible that deploying this will affect our production setup.
